circle.yml

```diff
@@ -0,0 +1,106 @@
+version: 2
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - xcode-10
+      - xcode-9
+
+step-library:
+  - &restore-cache
+      restore_cache:
+        keys:
+          - carthage-cache-mbstatic-v1-{{ .Environment.CIRCLE_JOB }}-{{ checksum "Cartfile.resolved" }}
+          - carthage-cache-mbstatic-v1- # used if checksum fails
+
+  - &save-cache
+      save_cache:
+        key: carthage-cache-mbstatic-v1-{{ .Environment.CIRCLE_JOB }}-{{ checksum "Cartfile.resolved" }}
+        paths:
+          - Carthage
+
+  - &prepare
+      run:
+        name: Prepare
+        command: |
+          git submodule sync
+          if (brew outdated | grep carthage > /dev/null); then brew upgrade carthage; fi
+          echo "foo" > ~/.mapbox
+
+  - &install-dependencies
+      run:
+        name: Install Dependencies
+        command: |
+          carthage bootstrap --platform ios --cache-builds --no-use-binaries
+          carthage bootstrap --platform tvos --cache-builds --no-use-binaries
+          carthage bootstrap --platform macos --cache-builds --no-use-binaries
+          carthage bootstrap --platform watchos --cache-builds --no-use-binaries
+
+  - &build-test-MapboxStatic-ios-11
+      run:
+        name: Build and Test MapboxStatic for iOS 11
+        command: |
+          xcodebuild -sdk iphonesimulator -project MapboxStatic.xcodeproj -scheme 'MapboxStatic iOS' -destination 'platform=iOS Simulator,OS=11.4,name=iPhone 6 Plus' clean build test
```

Comment:
The `build-test-MapboxStatic-macos` job pipes to `xcpretty`, but I notice none of these other steps do. Either way is fine — `xcpretty` can be easier to read, but might swallow useful information (unless you also [write](https://github.com/mapbox/mapbox-gl-native/blob/28bc4b2bd87fb15821b0aac911224577d4fe5a32/Makefile#L48) and [save](https://github.com/mapbox/mapbox-gl-native/blob/28bc4b2bd87fb15821b0aac911224577d4fe5a32/circle.yml#L420-L432) the raw log).